app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_toolbar.less

```diff
@@ -141,6 +141,14 @@
 //  Desktop
 //  _____________________________________________
 
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__xs) {
```

Comment:
@rodrigowebjump - I can not use xxs break point because problem occurs on screen size 330 or below 330 and nearest max screen break point is 480 which is xs. 
I need to set break point when screen size in decreasing, so need to use condition when screen size is between 320 to 330. Possible available is xs or 480